### ISSUE TITLE
Array Validation Question

### ISSUE THREAD

**Issue by benkiefer** [toranb/ember-cli-simple-validation#20]
We are trying to restrict multiple elements in an array from having the same value and ran into a problem with how the error messages show up in a strange way. I created a fork of the project and wrote a test showing the issue (all people must have a unique name). Would you mind taking a look?

https://github.com/benkiefer/ember-cli-simple-validation

In the test, we have 3 people, two of them have the name of Steve, and one is Sam and the inputs are delayed until form submission. When you submit, the errors show up. We are expecting that when you change one of the Steves to something else, that both of the errors hide.

Thoughts on how to accomplish this? Thanks @toranb!

**Comment by toranb** [comment]
@benkiefer thanks for writing as this looks like a great find! Internally when I watch the input change to Bob (from your test) I only see the 1 input validation fire (not all 3 like I'd expect because we _should be_ watching name on each model in the array)

https://github.com/toranb/ember-cli-simple-validation/blob/master/addon/mixins/validate.js#L63

Last night/tonight I'm looking into another PR on this project actually, so if I get time I'll see if I can take this for a spin also! I'll followup by weeks end at worst. Sorry for the delay -loads of OSS work this week :)

**Comment by toranb** [comment]
I threw up a pull request that solves this - let me know what you think and I could merge this/push up a patch to get you unblocked

https://github.com/toranb/ember-cli-simple-validation/pull/21
